### ISSUE TITLE
The QuickNav window should have a Go To HQ button

### ISSUE THREAD

**Issue by tothxa** [widelands/widelands#5957]
**Is your feature request related to a problem? Please describe.**
The Go To HQ function is much like the previous/next location and other landmark functions, but it's the only one of these that's not accessible from the Quick Navigation window. It should be added for consistency.

**Describe the solution you'd like**
Add a Go To HQ button to the QuickNav window.

**Additional context**
There's room for it in the top button row, but I don't know what would be the most logical order for the buttons with the HQ included. It currently has prev/add slot/next.
